src/checkstyle/checks.xml

```diff
@@ -184,16 +184,15 @@
 
         <module name="IllegalImport">
             <property name="illegalPkgs" value="
-                com.facebook.presto.hadoop.shaded,
```

Comment:
`com.facebook.presto.hadoop.shaded` is no longer matched

Reply:
Good catch, though that's intentional since everything now uses `$internal` instead of `shaded`.